bika/lims/content/sampletype.py

```diff
@@ -106,6 +121,47 @@
             visibile=False,
         )
     ),
+    RecordsField(
+        'AdmittedStickerTemplates',
+        subfields=(
+            'admitted',
+            'small_default',
+            'large_default',
+            ),
+        subfield_labels={
+            'admitted': _(
+                'Admitted stickers for the sample type'),
+            'small_default': _(
+                'Default small sticker'),
+            'large_default': _(
+                'Default large sticker')},
+        subfield_sizes={
+            'admitted': 6,
+            'small_default': 1,
+            'large_default': 1},
+        subfield_types={
+            'admitted': 'selection',
+            'small_default': 'selection',
+            'large_default': 'selection'
+                        },
+        subfield_vocabularies={
+            'admitted': sticker_templates(),
+            'small_default': '_small_default_voc',
+            'large_default': '_large_default_voc',
```

Comment:
No need of two different functions `_small_default_voc` and `_large_default_voc` for the same logic, just create a function `get_sticker_templates_vocabularies()` and use it instead.